src/controllers/user.controller.js

```diff
@@ -1 +1,27 @@
-exports.status = async (req, res, next) => res.json({ message: 'User endpoint is working.' })
\ No newline at end of file
+const User = require('../models/user.model')
+const errorHandler = require('../middlewares/error-handler')
+
+exports.status = async (req, res, next) => res.json({ message: 'User endpoint is working.' })
+
+exports.add_user = async (req, res, next) => {
```

Comment:
I feel like you should switch up the exporting style especially as the number of functions grow, not sure what @callmekungfu thoughts are on this. I feel like typically for nodejs you just have a module.exports at the end of the file no? Or treat it like a class and export the class?

Reply:
IMO, i think this is fine. If there end up being too many endpoints, I think I would separate from the type of Request: example.
```
users:
--> post
--> get
--> put
```